README.md

```diff
@@ -94,17 +94,20 @@ the external tree during the build process. To configure the system for build,
 choose any of the *defconfigs*  provided in this `BR2_EXTERNAL` tree. For example:
 
 ```bash
-For AM62X
-buildroot/ $ make BR2_EXTERNAL=../buildroot-external-TI ti_release_am62x_sk_defconfig
+For AM62X Linux
```

Comment:
```suggestion
# For AM62X Linux
```

Reply:
better to mark it as comment, for bash to recognize & highlight

Reply:
Updated @cshilwant